app/routes/api.js

```diff
@@ -3,37 +3,69 @@
 var express = require('express');
 var router = express.Router();
 
+var Account = require('../models/account');
 var Log = require('../models/log');
 
-// logs ======================================================================
-router.post('/logs', function(req, res) {
-    var userId = req.body.userId;
+module.exports = function(passport) {
+    // logs ======================================================================
+    router.post('/logs/', passport.authenticate('bearer', { session: false }), function(req, res) {
+        // force application/json content-type
+        if(req.headers["content-type"] !== 'application/json') {
```

Comment:
is this always lowercase and never capital (the value, not the key). if not, you should `.toLowerCase()` the value first